### ISSUE TITLE
Support adding attachment to message

### ISSUE THREAD

**Issue by JovicaSusa** [BookingSync/bookingsync-api#125]
We need to support adding attachments to message. We should use `add_attachment` endpoint in api v3 messages controller.

**Comment by thilonel** [comment]
Isn't flat a bit too low? That's for very simple stuff, this seems a bit more complex. Am I wrong?

**Comment by JovicaSusa** [comment]
Yeah, if flat is for very simple stuff then maybe tube is more appropriate.

**Comment by thilonel** [comment]
https://knowledgedatabase.happystay.com/en/tech-team-ticket-management ;)
